app/helpers/navigation_helper.rb

```diff
@@ -0,0 +1,5 @@
+module NavigationHelper
+	def cms_root_url(path)
```

Comment:
What do you think of doing something like:
```suggestion
	def cms_root_url(path='')
```
Then you can call `<%= link_to 'University of Alberta Libraries', cms_root_url %>`